src/material.cpp

```diff
@@ -60,8 +60,8 @@ namespace {
 
   // QueenMinorsImbalance[opp_minor_count] is applied when only one side has a queen.
   // It contains a bonus/malus for the side with the queen.
-  const int QueenMinorsImbalance[16] = { 
-    31, -8, -15, -25, -5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+  const int QueenMinorsImbalance[PIECE_NB] = {
```

Comment:
I would either keep it at 16 (cleanest) or alternatively change it to 13 (fine as well)